subnet/kube/kube.go

```diff
@@ -106,7 +107,7 @@ func NewSubnetManager(ctx context.Context, apiUrl, kubeconfig, prefix, netConfPa
 		return nil, fmt.Errorf("failed to read net conf: %v", err)
 	}
 
-	sc, err := subnet.ParseConfig(string(netConf))
+	sc, err := config.ParseConfig(string(netConf), config.KubeType)
```

Comment:
What about moving the "config.ParseConfig" to main.go? That way we have in main.go three steps:

1 - Configuration is parsed
2 - SubnetManager is started using the parsed config
3 - BackendManager is started using the parsed config + subnetManager to acquire the lease